gulpfile.js

```diff
@@ -245,7 +245,7 @@ gulp.task('clean:demo-cache', function() { return del('.publish/'); });
 
 gulp.task(
     'demo-server', ['generate-docs', 'generate-plunks'],
-    shell.task(['webpack-dev-server --port 9090 --config webpack.demo.js --hot --inline --progress']));
```

Comment:
Really? Weird. Shouldn't. We used that for hot refresh CSS, was handy for when we started to design the page.

You can leave it out.